tools/utils.py

```diff
@@ -130,6 +131,8 @@ def get_service_url(
         domain = get_domain(domain, target, namespace, profile)
         service_host = f"assisted-installer.{domain}"
         return to_url(service_host, disable_tls)
+    elif target == ONPREM_TARGET:
+        return "http://127.0.0.1:8090"
```

Comment:
There's already a `DEPLOY_TARGET` on assisted-test-infra with `podman-localhost` value. I think they should match to @rwsu [PR](https://github.com/openshift/assisted-test-infra/blob/c8ad10271943dd4ef55db4d444367123b2a89b77/README.md#L313-L325)

Honestly, I prefer "on-prem" instead of "podman-localhost" but I think they should be consistent.

Also, might want to support the code on [assisted-test-infra](https://github.com/openshift/assisted-test-infra/blob/50ef94b428244d84ee05bf199de24eb402904fce/discovery-infra/test_infra/utils.py#L521-L522) having `ASSISTED_SERVICE_HOST` env var.

Reply:
After a quick lookup, I see that there's already a `DEPLOY_TARGET` with `onprem` value on assisted-service. So we should change assisted-test-infra instead.